packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -105,7 +105,15 @@ export const processPatientDocumentRequest = async (
       );
     }
 
-    reportUsageCmd({ cxId, entityId: patientId, product: Product.medical });
+    const canReportUsage =
+      status === MAPIWebhookStatus.completed &&
+      documents &&
+      documents?.length > 0 &&
+      whType === "medical.document-download";
```

Comment:
It would be good to move this to a function and add some unit tests. It will be easy to validate multiple situations and add confidence that we're billing correctly.

Defs something we can improve on a follow-up ticket, though.

Reply:
https://github.com/metriport/metriport-internal/issues/1699